packages/kit-bg/src/services/ServicePrimeTransfer/e2ee/README.md

```
@@ -0,0 +1,6 @@
+# E2EE Server
+
+For the open source server implementation, please refer to:
+https://github.com/OneKeyHQ/e2ee-server
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)